server/src/tenant/metadata/tenant-migration/tenant-migration.entity.ts

```diff
@@ -5,16 +5,16 @@ import {
   PrimaryGeneratedColumn,
 } from 'typeorm';
 
-export type MigrationColumn = {
+export type TenantMigrationColumnChange = {
   name: string;
   type: string;
   change: 'create' | 'alter';
 };
 
-export type Migration = {
```

Comment:
I renamed it to avoid confusion with typeORM Migration, also the type name was too generic anyway

Reply:
agree!